lib/rbs_rails/active_record.rb

```diff
@@ -517,7 +518,46 @@ def clear_#{col.name}_change: () -> void
           sig
         end.join("\n")
         mod_sig << "\nend\n"
-        mod_sig << "include GeneratedAttributeMethods"
+        mod_sig << "include GeneratedAttributeMethods\n"
+        mod_sig
+      end
+
+      private def alias_columns
+        mod_sig = +"module GeneratedAliasAttributeMethods\n"
+        mod_sig << klass.attribute_aliases.map do |col|
+          old_col = klass.columns.find{_1.name == col[1]}
+          class_name = if enum_definitions.any? { |hash| hash.key?(col[0]) || hash.key?(col[0].to_sym) }
+                         'String'
+                       else
+                         sql_type_to_class(old_col.type)
+                       end
+          class_name_opt = optional(class_name)
+          column_type = old_col.null ? class_name_opt : class_name
+          sig = <<~EOS
+            def #{col[0]}: () -> #{column_type}
```

Comment:
I also thought about writing it as `alias alias_name name` , but I'm unsure which one is more appropriate.